app/src/main/java/net/squanchy/schedule/SchedulePageView.kt

```diff
@@ -122,6 +122,19 @@ class SchedulePageView @JvmOverloads constructor(
     private fun hasTypefaceSpan(text: CharSequence?) = if (text !is Spanned) false else text.hasTypefaceSpan()
 
     fun updateWith(schedule: Schedule, onEventClicked: (Event) -> Unit) {
+        progressbar.visibility = View.GONE
```

Comment:
⚠️  From your GIF this seems to happen immediately, and then we get the data displayed; is the gif not reflecting the code, or is the code not working as it should? The empty state should only appear after the data is actually fetched, and it's empty; while the first fetch is happening it should still show the progress bar

Reply:
The GIF was me starting with an empty state to demonstrate it 👅

Reply:
OH ok then 💃